roles/edpm_ovs/tasks/update.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Update openvswitch packages without stopping the service
+  become: true
+  tags:
+    - update
+    - ovs
+  # Call dnf via the shell module as the ansible dnf module does not
+  # support passing --setopt options. This is required in order to disable any
+  # OVS rpm scriptlets which would stop and disable OVS services during
+  # a standard update. Further info:
+  # https://github.com/openstack-k8s-operators/dev-docs/blob/main/ovs-update.md#rpm-packaging
+  ansible.builtin.shell: >
+    dnf update -y openvswitch --setopt=tsflags=noscripts
```

Comment:
should be use the values from "edpm_ovs_packages" variable instead of hardcoding here?

Reply:
Hmm , but if you read the description of `edpm_ovs_packages` "it could be used for pinning versions".  So if someone pins a version that could mess things up. I.e. `dnf update -y openvswitch<some_ver> `  Its also a list so order can change. 
I could perhaps define another variable but then it would always just be set to "openvswitch". So I dont see any benefit in that besides making these tasks more DRY. 
I am not going to make this change, let me know if you have further feedback here. thanks

Reply:
Yeap i saw that and that's the reason for asking clarification here. So if someone is explicitly pinning "edpm_ovs_packages" i think that means we should be sticking to that version whether it's fresh deploy or adoption or update. Having more variables just for this doesn't looks good. Is this assumption wrong?

Reply:
> So if someone pins a version that could mess things up. I.e. dnf update -y openvswitch<some_ver>

Could you please clarify what it will mess then? Won't it just attempt to update a package that is pinned? (so it will do a patch-level update, if present.)

>  Its also a list so order can change.

The list would require some massaging (`" ".join`?) but what does the order have to do here? I don't think `dnf` considers the order of packages in the command line; it has declarative/transactional semantics, no?

It seems like Yatin suggests to just reuse the existing variable; it sounds reasonable.

Reply:
btw if you don't reuse the existing variable, as Yatin suggests, then download-cache may miss your packages: https://github.com/openstack-k8s-operators/edpm-ansible/blob/e685241aa5bf3542401b5e1593582825ab9619c7/roles/edpm_ovs/tasks/download_cache.yml#L4

Reply:
See latest commit which address this